torch/csrc/distributed/c10d/TCPStore.hpp

```diff
@@ -9,6 +9,33 @@
 namespace c10d {
 namespace detail {
 
+// TCPStore essentially is a key-value pair storage we used for rendezvous
```

Comment:
TCPStore is a key-value store used by PyTorch mainly for distributed rendezvous, but for other purposes as well.